dask/dataframe/core.py

```diff
@@ -1067,6 +1148,24 @@ def loc(self):
 
         return _LocIndexer(self)
 
+    @property
+    def iloc(self):
+        """Purely integer-location based indexing for selection by position.
+
+        Only indexing the column positions is supported. Trying to select
+        row positions will raise a ValueError.
```

Comment:
This is no longer true.  We now have support for indexing rows, which is what allows us to filter on things like `"umi_counts">1000.0`.